### DISCUSSION TITLE
On Brave, favicons may not load on startup

### DISCUSSION THREAD

**Discussion by trlkly** [hrj/sloth#19] [Ideas]
I've found that, sometimes, the favicons (icons for each tab) don't load on startup if I use this extension in Brave. In Chrome, it seems to work fine. 

It doesn't happen all the time, so I suspect there's a race condition involved. And I note also that the "Discard tabs at startup" option in Auto Tab Discarder doesn't seem to have this problem. (However, it doesn't launch a new tab.) Perhaps looking at what they do could be useful. 

If nothing else, maybe add an option to delay startup of the extension by X milliseconds, then issue a "discard all tabs." That way any favicons will be guaranteed to load.

**Comment by hrj** [comment]
Basing functionality on a timed delay is not stable. It is unlikely that this extension will support such a workaround.

Moreover so, since it is a browser specific issue.

**Reply by trlkly** [reply]
That's understandable. I still would ask if it's possible to look at what Auto Tab Discarder does that avoids this problem. 

Of if anyone knows an extension that would just add the new tab page.
